index.html

```diff
@@ -30,7 +30,147 @@ <h2 class="connect">LET’S CONNECT</h2>
                 </ul>
         </div>
             </div>
+        </section>      
+        <section class="cardworks">
+            <div class="container">
+                <a href="#"> <img src="img/Snapshoot.png" alt="snapshoot"></a>
+                <h1 class="greet">Tonic</h1>
+                <div class="frame2">
+                    <div class="client">CANOPY</div>
+                    <div ><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
+                    <div class="role">Back End Dev</div>
+                    <div ><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
+                    <div class="role">2015</div>
+                </div>
+                    <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                   <p class="tag">html</p> 
+                   <p class="tag">css</p>
+                   <p class="tag">javascript</p>
+                </div>
+                <div class="button">
+                    <p class="seeproj">See Project</p>
+                </div>
+            </div>
+        </section>
+        <section class="cardworks">
+            <div class="container">
+                <a href="#"> <img src="img/Snapshoot2.png" alt="snapshoot"></a>
+                <h1 class="greet">Multi-Post</h1>
+                <h1 class="greet">Stories</h1>
+                <div class="frame2">
+                    <div class="client">CANOPY</div>
+                    <div ><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
+                    <div class="role">Back End Dev</div>
+                    <div ><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
+                    <div class="role">2015</div>
+                </div>
+                    <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                   <p class="tag">html</p> 
+                   <p class="tag">css</p>
+                   <p class="tag">javascript</p>
+                </div>
```

Comment:
- [Optional] It is best practice for HTML semantics to use an unordered list to list items. Kindly consider changing the list of technologies to an unordered list.